rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import babel from "rollup-plugin-babel";
+import typescript from "rollup-plugin-typescript2";
+import postcss from "rollup-plugin-postcss";
+// import external from "rollup-plugin-peer-deps-external";
+
+const packageJson = require("./package.json");
+
+export default {
+  input: "src/index.tsx",
+  output: [
+    {
+      file: packageJson.main,
+      format: "cjs",
+      sourcemap: true,
+    },
+    {
+      file: packageJson.module,
```

Comment:
try setting this up to  `packageJson.main` as well
(might fix Vite related issue)